frontend/src/component/project/Project/ProjectSettings/ChangeRequestConfiguration/ChangeRequestConfiguration.tsx

```diff
@@ -1,313 +1,67 @@
-                                            )
-                                        }
-                                        IconComponent={
-                                            KeyboardArrowDownOutlined
-                                        }
-                                        fullWidth
-                                    />
-                                </StyledBox>
-                            }
-                        />
-                    );
-                },
-                width: 100,
-                disableGlobalFilter: true,
-                disableSortBy: true,
-            },
-            {
-                Header: 'Status',
-                accessor: 'changeRequestEnabled',
-                id: 'changeRequestEnabled',
-                align: 'center',
-
-                Cell: ({ value, row: { original } }: any) => (
-                    <StyledBox data-loading>
-                        <PermissionSwitch
-                            checked={value}
-                            projectId={projectId}
-                            permission={UPDATE_PROJECT}
-                            inputProps={{ 'aria-label': original.environment }}
-                            onClick={onRowChange(
-                                original.environment,
-                                original.changeRequestEnabled,
-                                original.requiredApprovals
-                            )}
-                        />
-                    </StyledBox>
-                ),
-                width: 100,
-                disableGlobalFilter: true,
-                disableSortBy: true,
-            },
-        ],
-        []
+    const projectName = useProjectNameOrId(projectId);
+    const { hasAccess } = useContext(AccessContext);
+    const { isOss, uiConfig } = useUiConfig();
+    const isPro = !(
+        Boolean(uiConfig.versionInfo?.current.oss) ||
+        Boolean(uiConfig.versionInfo?.current.enterprise)
     );
+    usePageTitle(`Project change request – ${projectName}`);
 
-    const { getTableProps, getTableBodyProps, headerGroups, rows, prepareRow } =
-        useTable(
-            {
-                // @ts-ignore
-                columns,
-                data,
-                sortTypes,
-                autoResetGlobalFilter: false,
-                disableSortRemove: true,
-                defaultColumn: {
-                    Cell: TextCell,
-                },
-            },
-            useGlobalFilter
+    if (isOss() || isPro) {
+        return (
+            <PageContent
+                header={<PageHeader title="Change request configuration" />}
+                sx={{ justifyContent: 'center' }}
+            >
+                <ProFeatureTooltip
+                    title={'Enterprise feature'}
+                    origin={'Project Change Request Configuration'}
+                    center
+                >
+                    <>
+                        If you want to use{' '}
```

Comment:
I think we should align on an approach here in terms of text. I personally think something like "*feature name* requires a *plan level* version of Unleash. Check out ..." would work best. Something to double check with UX.

Reply:
One for when Nicolae is back I think